clients/csharp-dotnet-core/voice-assistant-test-generator/README.md

```diff
@@ -56,7 +56,7 @@ Each column's header should specify the schema of that property. Currently all v
 It is supported to have in depth objects. Currently if no braces are specified all properties are expected to be in the top level of the received activity. If you wish to specify depth simply add an opening brace to the object that has depth and closing brace to the last element.
 
 ```
-Utterance | type | ExpectedResponses | complexObject{ | subProperty1 | subObject2{ | subSubProperty1} | subProperty3}
```

Comment:
Was this intentional? Does not line up with sample below.

Reply:
updated the other line too. It is intentional since "type" isn't known header and should not come before "expectedResponses"